### PR TITLE
Update remove-users.md

### PR DESCRIPTION
**Pull request by tseward**
Fix for #2163. Updated `Remove-SPOUser` to show a non-external user claim.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/remove-users.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ For the steps to delete a user in the Microsoft 365 admin center, see [Delete a
...
+    > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall "SharePoint Online Management Shell." <br>On the Download Center page, select your language and then click the Download button. You'll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you're running the 64-bit version of Windows or the x86 file if you're running the 32-bit version. If you don't know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443). After the file downloads, run it and follow the steps in the Setup Wizard.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/remove-users.md:66]:
```suggestion
    > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall "SharePoint Online Management Shell."
    
    On the Download Center page, select your language and then click the Download button. You'll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you're running the 64-bit version of Windows or the x86 file if you're running the 32-bit version. If you don't know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443) After the file downloads, run it and follow the steps in the Setup Wizard.
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/remove-users.md:66]: Please remove the period after the link to "Which version of Windows operating system am I running?"

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**